src/test/java/org/openrewrite/java/testing/mockito/VerifyZeroToNoMoreInteractionsTest.java

```diff
@@ -0,0 +1,235 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.testing.mockito;
+
+import org.intellij.lang.annotations.Language;
+import org.junit.jupiter.api.Test;
+import org.openrewrite.DocumentExample;
+import org.openrewrite.InMemoryExecutionContext;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+import static org.openrewrite.maven.Assertions.pomXml;
+
+class VerifyZeroToNoMoreInteractionsTest implements RewriteTest {
+
+    @Language("xml")
+    private static final String POM_XML_WITH_MOCKITO_2 = """
+      <project>
+        <modelVersion>4.0.0</modelVersion>
+        <groupId>bla.bla</groupId>
+        <artifactId>bla-bla</artifactId>
+        <version>1.0.0</version>
+        <dependencies>
+          <dependency>
+              <groupId>org.mockito</groupId>
+              <artifactId>mockito-core</artifactId>
+              <version>2.17.0</version>
+              <scope>test</scope>
+          </dependency>
+        </dependencies>
+      </project>
+      """;
+
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec
+          .parser(JavaParser.fromJavaVersion()
+            .classpathFromResources(new InMemoryExecutionContext(), "mockito-core", "mockito-junit-jupiter", "junit-jupiter-api"))
+          .recipe(new VerifyZeroToNoMoreInteractions());
+    }
+
+    @Test
+    @DocumentExample
+    void shouldReplaceToNoMoreInteractions() {
+        //language=java
+        rewriteRun(
+          pomXml(POM_XML_WITH_MOCKITO_2),
+          java(
+                """
+            import org.junit.jupiter.api.extension.ExtendWith;
+            import org.mockito.junit.jupiter.MockitoExtension;
+            import org.junit.jupiter.api.Test;
+
+            import static org.mockito.Mockito.verifyZeroInteractions;
+
+            @ExtendWith(MockitoExtension.class)
+            class MyTest {
+                @Test
+                void test() {
+                    verifyZeroInteractions(System.out);
+                }
+            }
+            """, """
+            import org.junit.jupiter.api.extension.ExtendWith;
+            import org.mockito.junit.jupiter.MockitoExtension;
+            import org.junit.jupiter.api.Test;
+
+            import static org.mockito.Mockito.verifyNoMoreInteractions;
+
+            @ExtendWith(MockitoExtension.class)
+            class MyTest {
+                @Test
+                void test() {
+                    verifyNoMoreInteractions(System.out);
+                }
+            }
+            """));
+    }
+
+    @Test
+    void shouldNotReplaceToNoMoreInteractionsForImportOnly() {
```

Comment:
@timtebeek Not entirely sure about this one. As I used a method-matcher, the is no way you can match an import only. So I choose to ignore the unused-imports-case. If you don't agree, I'll change the code with another matcher (a `UsesType` perhaps?).

Reply:
Fine to not cover this one. If the import was unused they can fix that themselves.